server/middlewares/validation/internCompleteProfileSchema.js

```diff
@@ -1,37 +1,72 @@
 const Joi = require("@hapi/joi");
 
 const internCompleteProfileSchema = Joi.object({
+  // About you 1 (required for basic profile)
+  birthDate: Joi.date().required(),
+  gender: Joi.string().required(),
+  hometown: Joi.string().required(),
+  school: Joi.string().required(),
   profileImage: Joi.object({
     fileName: Joi.string().required(),
     isPrivate: Joi.boolean().default(false),
   }).required(),
-  verified: Joi.boolean().default(true),
+  interests: Joi.string().required(),
   bio: Joi.string().required(),
-  favouriteArticle: Joi.object({
-    title: Joi.string().required(),
-    author: Joi.string().required(),
-    link: Joi.string().required(),
-    description: Joi.string().required(),
-  }).required(),
-  jobTitle: Joi.string().required(),
-  organisation: Joi.string().allow(""),
+
+  // Other information 1 (required for booking)
+  organisation: Joi.string().required(),
+  useReasonAnswer: Joi.string().required(),
+  issueAnswer: Joi.string().required(),
+  storyAnswer: Joi.string().required(),
+  mentorDescribeAnswer: Joi.string().required(),
+
+  // about you 2 (required for basic profile)
   photoID: Joi.object({
     fileName: Joi.string().required(),
     isPrivate: Joi.boolean().default(true),
   }).required(),
+  hearAboutPressPadAnswer: Joi.string().required(),
+  phoneNumber: Joi.string().required(),
+
+  // Other information 2 (required for booking)
+  reference1: Joi.object({
+    name: Joi.string().required(),
+    email: Joi.string()
+      .email()
+      .required(),
+  }),
+  reference2: Joi.object({
+    name: Joi.string().required(),
+    email: Joi.string()
+      .email()
+      .required(),
+  }),
   offerLetter: Joi.object({
     fileName: Joi.string().required(),
     isPrivate: Joi.boolean().default(true),
   }),
-  reference1: Joi.object({
+  internshipOfficeAddress: Joi.string().required(),
+  emergencyContact: Joi.object({
     name: Joi.string().required(),
-    email: Joi.string().required(),
-  }).required(),
-  reference2: Joi.object({
-    name: Joi.string().required(),
-    email: Joi.string().required(),
-  }).required(),
+    email: Joi.required(),
```

Comment:
Is it ok to add a non-valid email here?

```suggestion
    email: Joi.string()
      .email()
      .required(),
```